book/update-theme.py

```diff
@@ -13,13 +13,33 @@
 INDEX_HBS_PORT = 443
 INDEX_HBS_PROTO = "GET"
 INDEX_HBS_PATH = "/repos/rust-lang/mdBook/contents/src/theme/index.hbs"
+INDEX_HBS_DATA = None
 INDEX_HBS_HEADERS = {
     "user-agent": "Update index.hbs for +https://github.com/rinja-rs/rinja",
 }
 
+ETHICALADS_THEME = r'''
+<style>
```

Comment:
Don't we have a style file or something? It's pretty bad for loading speed to keep the style in a `style` node.

Reply:
I append the content to a css file now during readthedocs' build process. More difficult to test now, but it should be sane.